### DISCUSSION TITLE
Welcome to Overmorrow Discussions!

### DISCUSSION THREAD

**Discussion by bmaroti9** [bmaroti9/Overmorrow#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cuhsy** [comment]
Hello can I help translate the app to Greek? Would love to contribute and have the Greek version on the app, and finally get to share it with friends and family

**Comment by bmaroti9** [comment]
Sure. Thanks for the offer!
You can go to the lib/languages.dart file and add your translations there, then create a pull request and i will merge.
We have already done this once for Polish so you can check how it was done, in the pull requests. 
If you have any question feel free to ask.

**Reply by cuhsy** [reply]
Hello just did this, let me know if I need to do anything else.

**Comment by cuhsy** [comment]
Don't forget to make sure the font supports Greek, since greek is a different dialect and very often is not supported by many fonts.

**Comment by bmaroti9** [comment]
Oh, I didn't think of that. I am currently using comfortaa, could you check if it supports your letters? If not could you recommend an alternate font that does, and looks similar.
